### PR TITLE
Add documentation on `What is BGP with EVPN`

### LINKED ISSUE #14: Create documentation "ELI5 - What is `BGP with EVPN` or `RFC-7432`"
link to [RFC-7432](https://datatracker.ietf.org/doc/html/rfc7432)

### LINKED ISSUE #37: ELI5 - What is the differences between type 2 and type 3 roads

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
closes #14, closes #37

### CODE REVIEW

File: docs/what-is-bgp-with-evpn.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## What is `BGP`
+
+`BGP` stand for `Border Gateway Protocol`, it's a *exterior gateway protocol* designed to exchange routing and reachability information on the Internet.
+
+`BGP` is classified as a `path-vector routing protocol`. It makes routing decisions based on paths, network policies or rule-sets configured by a network admin.
+
+## What is `BGP EVPN`
+
+`BGP EVPN` is the conbination of `BGP` and `EVPN` those the name.
```
Comment (Eviber) [inline docs/what-is-bgp-with-evpn.md:20]:
```suggestion
`BGP EVPN` is the combination of `BGP` and `EVPN`, thus the name.
```

File: docs/what-is-bgp-with-evpn.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+`EVPN` for `Ethernet VPN` is a technologie for carrying layer 2 Ethernet traffic in a VPN using WAN[^wan] protocols.
+`EVPN` technologie inclue Ethernet over `MPLS`[^mpls] and Ethernet over `VXLAN`[^vxlan].
+
+[^wan]: Wide Area Network
+[^mpls]: Multiprotocol Label Switching
+[^vxlan]: Virtual Extensible Local Area Network
+
+## What is `BGP`
+
+`BGP` stand for `Border Gateway Protocol`, it's a *exterior gateway protocol* designed to exchange routing and reachability information on the Internet.
```
Comment (Eviber) [inline docs/what-is-bgp-with-evpn.md:14]:
```suggestion
`BGP` stand for `Border Gateway Protocol`, it's an *exterior gateway protocol* designed to exchange routing and reachability information on the Internet.
```

File: docs/what-is-bgp-with-evpn.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# What is `BGP with EVPN`
+
+## What is `EVPN`
+
+`EVPN` for `Ethernet VPN` is a technologie for carrying layer 2 Ethernet traffic in a VPN using WAN[^wan] protocols.
+`EVPN` technologie inclue Ethernet over `MPLS`[^mpls] and Ethernet over `VXLAN`[^vxlan].
```
Comment (Eviber) [inline docs/what-is-bgp-with-evpn.md:6]:
```suggestion
`EVPN` technologies include Ethernet over `MPLS`[^mpls] and Ethernet over `VXLAN`[^vxlan].
```

File: docs/what-is-bgp-with-evpn.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# What is `BGP with EVPN`
+
+## What is `EVPN`
+
+`EVPN` for `Ethernet VPN` is a technologie for carrying layer 2 Ethernet traffic in a VPN using WAN[^wan] protocols.
```
Comment (Eviber) [inline docs/what-is-bgp-with-evpn.md:5]:
```suggestion
`EVPN` for `Ethernet VPN` is a technology for carrying layer 2 Ethernet traffic in a VPN using WAN[^wan] protocols.
```

File: docs/what-is-bgp-with-evpn.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+`BGP` stand for `Border Gateway Protocol`, it's a *exterior gateway protocol* designed to exchange routing and reachability information on the Internet.
+
+`BGP` is classified as a `path-vector routing protocol`. It makes routing decisions based on paths, network policies or rule-sets configured by a network admin.
+
+## What is `BGP EVPN`
+
+`BGP EVPN` is the conbination of `BGP` and `EVPN` those the name.
+It's a protocol that can be used with VXLAN.
+With `BGP EVPN` ours `VTEPs`[^vtep] with publish 2 type of routes:
```
Comment (Eviber) [inline docs/what-is-bgp-with-evpn.md:22]:
```suggestion
With `BGP EVPN` our `VTEPs`[^vtep] will publish 2 type of routes:
```

**Review by Ragarnoy — APPROVED** [review]

**Review by Eviber — APPROVED** [review]
